src/debug.h

```diff
@@ -12,9 +12,11 @@
 
 #include "cpu.h"
 #include <chrono>
+#define FMT_HEADER_ONLY
+#include "3rdparty/fmt/format.h"
```

Comment:
as I would expect more places to start using `fmt`, maybe we should make a `core/fmt_type.hpp` or what-ever that does this header-dance for us? Just to make sure we don't screw that up over the years :P

Reply:
Alternatively, the define could be in `stdafx.h`, then it is always set whenever `fmt` is used.